src/routes.js

```diff
@@ -28,7 +29,8 @@ export default function Routes() {
                 <Route path="/forms" component={ComingSoon} />
                 <Route path="/messages" component={ComingSoon} />
                 <Route path="/pages" exact component={Pages} />
-                <Route path="/pages/users" component={Users} />
+                <Route path="/pages/users" exact component={Users} />
+                <Route path="/pages/users/detail" exact component={UserDetail} />
```

Comment:
For now use ```path="/pages/user/details```, so you can remove the exact in the above route.
When connect with redux the path will be:
```/pages/user/:uid```
then the uid wilcard will be accessed in this way:
```const { uid } = this.props.match.params;```
then en didMount we fetch the user details by the uid